test/game_collection_test.rb

```diff
@@ -23,6 +23,28 @@ def test_it_can_create_games_from_csv
     assert_equal 3, @game.home_goals
   end
 
+  def test_it_can_calculate_average_goals_per_game
+    expected = 4.31
```

Comment:
Looks good. 4.31 just needs to be moved to next line

Reply:
on it